launch/load_tif_launch.xml

```diff
@@ -3,9 +3,13 @@
     <arg name="location" default="sargans"/>
     <node pkg="tf2_ros" exec="static_transform_publisher" name="world_map" args="--frame-id world --child-frame-id map"/>
 
-    <node pkg="grid_map_geo" exec="test_tif_loader" name="test_tif_loader" namespace="grid_map_geo" output="screen">
-        <param name="tif_path" value="$(find-pkg-share grid_map_geo)/resources/sargans.tif"/>
-        <param name="tif_color_path" value="$(find-pkg-share grid_map_geo)/resources/sargans_color.tif"/>
+    <node pkg="grid_map_geo" exec="map_publisher" name="map_publisher" namespace="grid_map_geo" output="screen">
```

Comment:
Shouldn't we keep the tif loader here?

Reply:
Yea. I was curious why have both python and XML launch files for TIF, and do you want both for the VRT loader?

Reply:
No real preference :) whatever is more convenient

Reply:
Fixed.